system/shaders/GLES/2.0/gles_shader_texture_noalpha.frag

```diff
@@ -0,0 +1,27 @@
+/*
+ *  Copyright (C) 2019 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#version 100
+
+precision mediump float;
+uniform sampler2D m_samp0;
+varying vec4 m_cord0;
+
+void main ()
+{
+  vec4 rgb;
```

Comment:
Suggestion for all of `main`. Simplify the code and not read the 4th component of the texture since it's not used:
```glsl
void main ()
{
  vec3 rgb;
   rgb = texture2D(m_samp0, m_cord0.xy).rgb;

 #if defined(KODI_LIMITED_RANGE)
  rgb *= (235.0 - 16.0) / 255.0;
  rgb += 16.0 / 255.0;
#endif

   gl_FragColor = vec4(rgb, 1.0);
}
```

Reply:
I just tried this and games work, but now GUI textures are having the invisible problem

Reply:
Hm weird, it should be functionally the same.

Reply:
Must have been stale build files, because I retried your suggestion and found a typo. With the typo fixed, everything works. I'm adding your suggestion to the PR.

Reply:
> I just tried this and games work, but now GUI textures are having the invisible problem

I'm seeing a similar issue with OSMC builds.  Invisible  logos under MyOSMC.  Tested using the latest commit as well.